core/iwasm/interpreter/wasm.h

```diff
@@ -1181,6 +1181,55 @@ wasm_string_equal(const char *s1, const char *s2)
     return strcmp(s1, s2) == 0 ? true : false;
 }
 
+/* FIXME: move to aot_loader.c if only required by aot */
+/*
+ * Whether a type is known and valid
+ */
+inline static bool
+is_valid_value_type(uint8 value_type)
+{
+    if (value_type == VALUE_TYPE_VOID)
+        return true;
+
+    if (value_type == VALUE_TYPE_I32 || value_type == VALUE_TYPE_F32
+        || value_type == VALUE_TYPE_ANY)
```

Comment:
VALUE_TYPE_ANY is wamr's internal type for the integrity check in wasm loader, had better not add it here?